packages/node-resolve/rollup.config.js

```diff
@@ -1,15 +1,23 @@
-import json from '@rollup/plugin-json';
-
 import { emitModulePackageFile } from '../../shared/rollup.config';
 
 import pkg from './package.json';
 
 export default {
   input: 'src/index.js',
-  plugins: [json()],
+  plugins: [
+    {
+      name: 'external-package-json',
+      resolveId(source) {
+        if (source.endsWith('package.json')) {
+          return { id: '../../package.json', external: true };
```

Comment:
for posterity we should add a comment for why it's looking in the parent-parent directory